### PR TITLE
integrate clave

### PR DESCRIPTION
**Pull request by AndrewCLu**

### CODE REVIEW

File: src/components/profileFormSteps/index.tsx [hunk 1]
```diff
@@ -203,6 +215,23 @@ const ProfileForm = ({
...
       </div>
+      {claveInfo && (
+        <div className="flex flex-col gap-7">
+          <span className="text-gray-12 text-sm font-light">Clave</span>
+          {claveInfo.claveWalletAddress ? (
+            <span className="text-gray-12 text-sm font-light">{`Wallet Address: ${claveInfo.claveWalletAddress}`}</span>
+          ) : (
+            <Button
+              type="button"
+              onClick={() => window.open(claveInfo.claveInviteLink, "_blank")}
```
Comment (Clee681) [inline src/components/profileFormSteps/index.tsx:226]: lfgggg

File: src/lib/server/clave.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  userEmail: string,
+  userClaveInviteCode: string
+) => {
+  const jwt = signJwt(userEmail);
+
+  return `getclave.io/link/summon?signature=${jwt}&waitlist=${userClaveInviteCode}`;
+};
+
+import jwt from "jsonwebtoken";
+import dayjs from "dayjs";
```
Comment (Clee681) [inline src/lib/server/clave.ts:15]: shall we move import to top of file or nah

**Review by Clee681 — APPROVED** [review]
full send 🚀

File: src/pages/api/user/get_balance.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    const user = await prisma.user.findUnique({
+      where: { id: userId },
+    });
+    if (!user) {
+      return res.status(400).json({ error: "Invalid user id" });
+    }
+
+    const buidlBalance = await getUserBuidlBalance(user.id.toString());
+    const claveWalletAddress = user.claveWallet ? user.claveWallet : undefined;
+    const claveInviteLink = await getClaveInviteLink(
```
Comment (RiverRuby) [inline src/pages/api/user/get_balance.ts:38]: generates a new invite link every time we check balance? seems incorrect

**Review by RiverRuby — APPROVED** [review]
LGTM if new invite link bug is fixed or if it is necessary for some reason i dont konw

File: src/lib/server/clave.ts (hunk 2 above)
Comment (AndrewCLu) [reply src/lib/server/clave.ts]: great catch!

File: src/pages/api/user/get_balance.ts (hunk 3 above)
Comment (AndrewCLu) [reply src/pages/api/user/get_balance.ts]: it is incorrect. fixed

**Pull request merged**